.travis.yml

```diff
@@ -0,0 +1,37 @@
+sudo: required
+
+services:
+  - docker
+
+language: java
+
+matrix:
+  fast_finish: true
+  include:
+    - os: linux
+      jdk: oraclejdk10
+#    - os: osx
+#      osx_image: xcode8
```

Comment:
Why comment out mac?

Reply:
https://github.com/wpilibsuite/shuffleboard/blob/master/.travis.yml#L13

Reply:
Enable it for now.